src/main/scala/dsptools/numbers/algebra_types/helpers/Sign.scala

```diff
@@ -69,6 +69,7 @@ object Sign {
     override def sign(a: Sign): Sign = a
     def signum(a: Sign): ComparisonBundle = ComparisonHelper(a.zero, a.neg)
     def abs(a: Sign): Sign = if (a == Negative) Positive else a
+    def context_abs(a: Sign): Sign = if (a == Negative) Positive else a
```

Comment:
Im not sure that's necessay... (also only included b/c Paul wanted Spire "Sign", but I can't think of a use case...)